Initial draft of design team goals (#5555)
sourcegraph/handbook: content/departments/engineering/design/goals/index.md

# Design team goals

The design team goals help us align and prioritize our focus as a team. These team-level goals cascade from the overall company [product vision and
strategy](../../index.md#product-vision-and-strategy). Additionally, each member of the design team
will define their own individual goals based on the career ladder, their product team objectives, and/or the design team goals.

## Design team goals for Q4, FY23

### Urgent and important

#### Theme: Win the top 100 organizations

1. **Objective**: Remove blocks to adoption related to UI/UX for 2XL customers.
   - Key Result: Provide design support for all the key issues determined to be in scope by the strategic readiness effort.

>      audit.

Author: ```suggestion
```

2. **Objective**: Get closer to our users.
   - Key Result: Each member of the design team participates in at least 1
     sales, AE, or CE session per week.

>    - Key Result: Each member of the design team has at least 14 hours of

Reviewer: ```suggestion
   - Key Result: Each member of the design team has at least 10 hours of
```

## (Draft) Design team goals for Q4

### Urgent and important

#### Theme: Win the top 100 organizations

1. **Objective**: Remove blocks to adoption related to UI/UX for 2XL customers.
   - Key Result: Provide design support for all the key issues determined to be in scope by the strategic readiness effort.
     audit.
2. **Objective**: Get closer to our users.
   - Key Result: Each member of the design team participates in at least 1
     sales, AE, or CE session per week.
   - Key Result: Each member of the design team has at least 14 hours of
     user time per quarter.

#### Theme: Drive expansion in existing customers

1. **Objective**: Identify opportunities to improve expansion in 2XL customers.
   - Key Result: We have identified specific opportunities to improve
     expansion within at least two 2XL customers.
   - Key Result: Specific projects have been
     prioritized with reference to the opportunities we identified.
   - Key Result: 10% MAU growth is observed in customers where projects are executed.

### Non-urgent and important

#### Theme: Levelling up the design team

1. **Objective**: Level up storytelling skills.

>    - Key Result: Each member of the design team shares two #progress announcements that exhibt strong storytelling skills.

Reviewer: ```suggestion
   - Key Result: Each member of the design team shares two #progress announcements that exhibit strong storytelling skills.
```

# Design team goals

We have team-level design team goals to help us align and prioritize our
focus as a team for the remainder of Q4.

Our team-level goals cascade from the overall company [product vision and
strategy](../../index.md#product-vision-and-strategy). Additionally, each member of the design team
will define their own individual goals based on the career ladder, their product team objectives, and/or the design team goals.

## (Draft) Design team goals for Q4

### Urgent and important

#### Theme: Win the top 100 organizations

1. **Objective**: Remove blocks to adoption related to UI/UX for 2XL customers.

>    - Key Result: Eliminate all the UI/UX-related must-haves from the strategic scale

Reviewer: ```suggestion
   - Key Result: Provide design support for all the key issues determined to be in scope by the strategic readiness effort.
```

# Design team goals

We have team-level design team goals to help us align and prioritize our

> focus as a team over the following 3-6 months.

Reviewer 2: 3-6 months is a pretty varying range in timespan. Is there a reason for this specific timespan? Did we want to align to quarters?

Reviewer: Agreed. These goals were meant to align to the end of Q4.

> focus as a team over the following 3-6 months.

Reviewer: ```suggestion
focus as a team for the remainder of Q4.
```

> Our team-level goals cascade from the overall company [product vision and

Reviewer 2: I think some clarification on the connection btw the company product vision/strategy and the (design team) themes would be helpful here! At first, I thought the themes were equivalent to the company product strategy, but it seems they're themes derived from the product strategy but more design team specific.

strategy](../../index.md#product-vision-and-strategy), and each member of the design team
uses these to define their own individual goals.

> ## (Draft) Design team goals for the next 3-6 months

Reviewer: ```suggestion
## (Draft) Design team goals for Q4
```

### Urgent and important

#### Theme: Win the top 100 organizations

1. **Objective**: Remove blocks to adoption related to UI/UX for 2XL customers.

>    - Key Result: Eliminate all the UI/UX-related must-haves from the scale

Reviewer: ```suggestion
   - Key Result: Eliminate all the UI/UX-related must-haves from the strategic scale
```

     audit.

>    - Key Result: Design as a function contributes to closing a deal for a

Reviewer 2: How might we measure this? It could be closing out an issue(s) that is an improvement/feature request that is directly requested by a 2XL customer, but this would be reactive and requires that a customer preemptively requests something that is then tagged as needing design.

Author: This is intended to be directly tied to the strategic work underway. Assuming we have a specific list of issues (@AlicjaSuska), then measuring it would be tied to addressing these issues.

I'm thinking that as an objective/key result, it should more specifically align it to the strategic effort?

Reviewer 3: One additional fly-by note: Yes, @AlicjaSuska's work is absolutely critical path for strategic. In addition,  every team across the org is working to increase the strategic readiness of their features and design support will be necessary alongside each of those teams as well. For example, BC team will certainly need thoughtful design as they work on the key issues outlined in their [strat readiness doc](https://docs.google.com/document/d/1vXAnhui-AGBk2aar4pXt-uTuTIQfZU5LFFtrMAU2e4s/edit#bookmark=id.9gi05kl6on12) that fall outside of Alicja's central work.

Reviewer 4: Maybe we could include this in the first KR:
Key Result: Eliminate all the UI/UX-related must-haves from the scale audit.
Key result: Provide design support for all the key issues outlined in the strategic readiness doc for respective teams.

Reviewer: I altered this with a slightly edited version of Alicja's comment and will remove the last KR as I think it's a bridge too far and would be hard to measure. If anyone disagrees, feel free to bring it back.

     2XL customer.
2. **Objective**: Get closer to our users.
   - Key Result: Each member of the design team participates in at least 1
     sales, AE, or CE session per week.

>    - Key Result: Each member of the design team has at least 14 hours of

Reviewer: Note, I set the time goal to 10 hours of user-facing time counting 7 weeks until Jan 31 (minus 2 weeks due to Thanksgiving and Christmas), 45 minutes per session, and 2 sessions per week.

>    - Key Result: Each member of the design team has at least `${x}` hours of

Reviewer: ```suggestion
   - Key Result: Each member of the design team has at least 14 hours of
```

>    - Key Result: Each member of the design team has at least `${x}` hours of

Reviewer: We need to determine what ${x} is. I suggest we use the goal of 1 hour per week (about 2 sessions), and should compensate for the holidays. 

We also need to determine what 'user time' means. I suggest:

> Generative or evaluative research sessions with both internal and external users.

Author: I also believe at least 1 hour per week is reasonable—perhaps our intent is "per _typical_ week" (which can be defined with an asterisk)?

For "user time," I'd suggest starting with both internal and external users, but in the following quarter, evolving to prefer external users.

How do we consider things like async usertesting.com sessions? Are they part of this time?

Reviewer: I agree on moving primarily toward external. I thnk that user testing session reviews should count toward this time. That said, we'll want to make sure this isn't the only type of testing conducted.

Reviewer 2: Does time spent synthesizing and tagging this information in Dovetail also part of this time? I think it's important to mention/acknowledge that it takes just as much time to pull out the insights from user interviews into Dovetail properly (I'm already guilty of being behind on this myself 😅 ). I'm not necessarily advocating for fewer interviews, but if we mean 1 hour a week _conducting_ user interviews, realistically that's another half hour to another hour tagging and pulling out insights in Dovetail.

Author: No, I think this is a measure of time spend with users. I think synthesizing is something we'll get more efficient at as we go.

>      user time per `${cycle}`.

Reviewer: ```suggestion
     user time per quarter.
```

Aligning this to the intended cadence.

#### Theme: Drive expansion in existing customers

1. **Objective**: Identify opportunities to improve expansion in 2XL customers.
   - Key Result: We have identified specific opportunities to improve
     expansion within at least two 2XL customers.
   - Key Result: Specific projects have been
     prioritized with reference to the opportunities we identified.
   - Key Result: 10% MAU growth is observed in customers where projects are executed.

### Non-urgent and important

#### Theme: Levelling up the design team

1. **Objective**: Identify and follow through on opportunities for professional
   growth.
   - Key Result: Identify and carry out a professional development activity
     related to the function of design.
   - Key Result: Identify and carry out a professional development activity
     related to the function of engineering.
2. **Objective**: Level up storytelling skills.
   - Key Result: Each member of the design team shares a #progress announcement.

>    - Key Result: Company-wide engagement in design review increases.

Reviewer 2: I feel as if this is more an Objective that the team should aim to accomplish, and one of the KRs could be that design team members share to #progress. Another potential KR (pulling from a design offsite) is using some form of media (video snippets, user interview quotes, etc.) in a #progress post or team discussion.

Reviewer: I tried to capture this by adding: "that exhibit strong storytelling skills."
